dataplane/oauth/oauth_client.go

```diff
@@ -21,6 +22,10 @@ type Cloudbreak struct {
 	Cloudbreak *apiclient.Cloudbreak
 }
 
+type Dataplane struct {
```

Comment:
Why we need this wrapper struct?

Reply:
We want to uniquely identify Dataplane client so created this. Any recommendation on how this can be done

Reply:
Does `authapiclient.Dataplane` not identify uniquely enough?

Reply:
basically i followed the same convention as used by Cloudbreak.
```
type Cloudbreak struct {
	Cloudbreak *apiclient.Cloudbreak
}

type Dataplane struct {
	Dataplane *authapiclient.Dataplane
}
```